web_monitoring/db.py

```diff
@@ -12,6 +12,7 @@
 
 
 DEFAULT_URL = 'https://api.monitoring.envirodatagov.org'
+DEFAULT_TIMEOUT = 30.5
```

Comment:
I saw you used 30.5 in the [WaybackRecordsWorker](https://github.com/edgi-govdata-archiving/web-monitoring-processing/blob/a05fd623f5b983db84d4f2370197a61bd4276153/web_monitoring/cli/cli.py#L235), so I went with that (guessing you chose it based on the recommendation to use something slightly bigger than a multiple of 3 over in [requests docs](https://requests.readthedocs.io/en/master/user/advanced/#timeouts)).

Reply:
Ahhh, we use a super long timeout for Wayback because it is *very* slow for pages that aren’t accessed frequently. This is probably fine, though.